specification/dimensions/usagequantity.md

```diff
@@ -0,0 +1,31 @@
+# Usage Quantity
```

Comment:
The name should be updated to QuantityInUsageUnit to be consistent with other names.